packages/keyring-api/src/api/account.ts

```diff
@@ -45,6 +45,11 @@ export type KeyringAccountType =
   | `${BtcAccountType.P2wpkh}`
   | `${SolAccountType.DataAccount}`;
 
+/**
+ * Account ID (UUIDv4).
+ */
+export const AccountIdStruct = UuidStruct; // Alias for better naming purposes.
```

Comment:
I'm ok with this definition being here, but we could consider having it in `keyring-utils` (I have no preference, just raising the point)

Reply:
At least it would be closer to `UuidStruct` so I'm ok with that.

I just thought that `AccountId` types being part of the `keyring-api` made sense too!